### DISCUSSION TITLE
Angular 17 support

### DISCUSSION THREAD

**Discussion by spike1292** [Sitecore/jss#1721] [Ideas]
### Is your suggestion related to a problem? Please describe.

Would like to use the new angular 17 new Built-in control flow

### Describe the solution you'd like

sitecore-jss-angular published in APF 17.

### Additional information

Could also have performance gains

**Comment by yavorsk** [comment]
Hey @spike1292 , we've created an item in our system for upgrade to Angular 17

**Comment by art-alexeyenko** [comment]
@spike1292 a bit of a delayed update, but JSS has been updated to use Angular 17 in [JSS 22.1](https://github.com/Sitecore/jss/releases/tag/v22.1.0)

Please let us know if you tried it already and if you face any issues.

**Reply by art-alexeyenko** [reply]
Just a note, Angular 17 is now available in JSS 21.8 as well.

**Reply by spike1292** [reply]
I already tested it and its working for us!
